ui/components/component-library/icon/icon.stories.js

```diff
@@ -207,6 +207,17 @@ export const Size = (args) => (
     <Icon {...args} size={SIZES.MD} />
     <Icon {...args} size={SIZES.LG} />
     <Icon {...args} size={SIZES.XL} />
+    <Text
+      as="span"
+      variant={TEXT.HEADING_LG}
+      display={DISPLAY.FLEX}
+      alignItems={ALIGN_ITEMS.CENTER}
+    >
+      <Icon {...args} size={SIZES.AUTO} marginRight={1} />
+      <span>
+        Size auto also exists and inherits the font-size of the parent element
+      </span>
+    </Text>
   </Box>
```

Comment:
Sorry should have given the full suggestion for this 

```
export const Size = (args) => (
  <>
    <Box
      display={DISPLAY.FLEX}
      alignItems={ALIGN_ITEMS.BASELINE}
      gap={1}
      marginBottom={4}
    >
      <Icon {...args} size={SIZES.XXS} />
      <Icon {...args} size={SIZES.XS} />
      <Icon {...args} size={SIZES.SM} />
      <Icon {...args} size={SIZES.MD} />
      <Icon {...args} size={SIZES.LG} />
      <Icon {...args} size={SIZES.XL} />
    </Box>
    <Text
      as="span"
      variant={TEXT.BODY_MD}
      display={DISPLAY.FLEX}
      alignItems={ALIGN_ITEMS.CENTER}
    >
      <Icon {...args} size={SIZES.AUTO} marginRight={1} />
      <span>
        Size auto also exists and inherits the font-size of the parent element
      </span>
    </Text>
  </>
);
```

<img width="918" alt="Screen Shot 2022-09-28 at 12 26 36 PM" src="https://user-images.githubusercontent.com/8112138/192884069-901bd36f-2f7d-4298-96c7-ef2ba5e81007.png">